src/com/lftechnology/java/training/srijan/basics/football/ManchesterUnitedStats.java

```diff
@@ -0,0 +1,229 @@
+
+				if (inputScore.hasNext()) {
+					scores = inputScore.nextLine();
+					this.fixtureWithScore[++arrayCounter] = scores;
+					LOGGER.info(gamesFixtures[team + 1] + " score: ");
+					this.fixtureWithScore[++arrayCounter] = gamesFixtures[team + 1];
+					scores = inputScore.nextLine();
+					this.fixtureWithScore[++arrayCounter] = scores;
+				}
+				team = team + 2;
+				arrayCounter++;
+			}
+
+		} catch (NoSuchElementException ex) {
+			LOGGER.info("Error Message:" + ex.getMessage());
+		} catch (ArrayIndexOutOfBoundsException exception) {
+			LOGGER.info("Exception Message:" + exception.getMessage());
+		} finally {
+			displayFixturesAndScores(numberOfMatches);
+			inputScore.close();
+		}
+	}
+
+	/**
+	 * <p>
+	 * Displays Fixtures And Scores of GameWeek Matches
+	 * </p>
+	 * 
+	 * @param fixtureAndScores
+	 *            {@link String}
+	 * @param numberOfMatches
+	 *            {@link Integer}
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+	public void displayFixturesAndScores(int numberOfMatches) {
+		int counter = 0;
+		LOGGER.info(clubName + "Fixtures And Scores");
+		for (int i = 0; i < numberOfMatches; i++) {
+			LOGGER.info(this.fixtureWithScore[counter] + " " + this.fixtureWithScore[counter + 1] + " "
+					+ this.fixtureWithScore[counter + 2] + " " + this.fixtureWithScore[counter + 3]);
+			counter = counter + 4;
+		}
+
+		checkForManUtdInString(numberOfMatches);
+	}
+
+	/**
+	 * <p>
+	 * Checks for Manchester United in an array
+	 * </p>
+	 * <p>
+	 * Calculates goal scored
+	 * </p>
+	 * 
+	 * @param numberOfMatches {@link Integer}
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+	public void checkForManUtdInString(int numberOfMatches) {
+		int positionHome = 0;
+		int positionAway = 2;
+		int scored = 0;
+		int against = 0;
+		int goalScored = 0;
+		int goalConceded = 0;
+		for (int i = 0; i < numberOfMatches; i++) {
+			if (clubName.equals(fixtureWithScore[positionHome])) {
+				scored = Integer.parseInt(fixtureWithScore[positionHome + 1]);	
+			}
+
+			else if (clubName.equals(fixtureWithScore[positionAway])) {
+				scored = Integer.parseInt(fixtureWithScore[positionAway + 1]);
+			}
+			goalScored = goalScored + scored;
+			against = calculateGoalsConceded(numberOfMatches, positionHome, positionAway);
+			goalConceded = goalConceded + against;
+			positionHome = positionHome + 4;
+			positionAway = positionAway + 4;
+		}
+		LOGGER.info("Goals Scored " + goalScored);
+		LOGGER.info("Goal Conceded " + goalConceded);
+		displayResults(numberOfMatches);
+		
+	}
+	
+	
+	
+	/**
+	 * <p>Calculates Number of Goals Conceded</p>
+	 * @param numberOfMatches {@link Integer}
+	 * @param positionHome {@link Integer}
+	 * @param PositionAway {@link Integer}
+	 * @return totalAgainst {@link Integer}
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+	public int calculateGoalsConceded(int numberOfMatches, int positionHome, int positionAway) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "numberOfMatches". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1172)